### DISCUSSION TITLE
Dynamic Server Error

### DISCUSSION THREAD

**Discussion by chris-nowicki** [vercel/next.js#48835] [Help]
### Summary

I have recently been using NextJS 13 app router in conjunction with Vercel Cron Jobs.  I have been using this since 13.2.4 without issues.  When I updated to 13.3.1 I now get the following errors during build.  Not sure if this is because it is still in beta or something changed that I am unaware of.

![Screenshot 2023-04-25 at 2 52 36 PM](https://user-images.githubusercontent.com/102450568/234374930-8703e244-1a13-4233-acc5-5c0e80821c00.png)

Note that the scripts for the cron-jobs in /app/cron/** work when I run them manually in dev mode locally.  Any help appreciated!

### Additional information

```js

```

**Comment by alexknipfer** [comment]
Did you figure out the cause of this error? I’m seeing this same error in the latest 13.4.1 where app router is now stable

**Comment by chris-nowicki** [comment]
I did not figure it out.  I had a support ticket in with Vercel and it has been resolved for the last two releases.  I just checked 13.4.1 and do not have the dynamic error on my end anymore.  Are you seeing this with your cron jobs?
